web-client/src/app/views/two-factor-authentication/two-factor-authentication.page.ts

```diff
@@ -18,6 +21,8 @@ export class TwoFactorAuthenticationPage implements OnInit {
   isOpening = false;
   registrationForm: FormGroup;
   walletId: string | undefined;
+  otpPhoneNumber: string | undefined;
+  subscription: Subscription | undefined;
```

Comment:
are you still using this?

Reply:
No we are not using subscription variable.
I've removed it.